pkg/api/errors/metdata.go

```diff
@@ -0,0 +1,34 @@
+package errors
+
+import (
+	"fmt"
+    "net/http"
```

Comment:
looks like you need to run: `make lint` locally

Reply:
Oh... thanks